### DISCUSSION TITLE
Star and watch count should be visible from every tab of a repository.

### DISCUSSION THREAD

**Discussion by erusev** [community/community#70529] [Repositories]
### Select Topic Area

Product Feedback

### Body

This is how it used to work before the latest redesign.

Use case: 
You land on the Discussions or the Issues page of a repository, and you want to get a sense of how popular this repository is.

**Comment by tophf** [comment]
+1. The new UX is inconvenient and counter-intuitive: why do I need to click the `code` tab in order to star or watch the repo? There's no way to invoke these features via the commands palette either.

**Comment by erusev** [comment]
Another use case:
I land on the `Releases` page of a repo and I decide that I want to subscribe to future releases.
Why do I have to go to the `Code` tab to do so?
